### ISSUE TITLE
Broken image files crash (white screen)

### ISSUE THREAD

**Issue by valtusovs** [desktop/desktop#8028]
## Description

When a commit corrupts an image file, whenever the corrupt file is clicked to check for diffs (in staged view or in commits view), the whole UI becomes fully white.

## Version

* GitHub Desktop: 2.1.0

* Operating system: Microsoft Windows [Version 10.0.17763.557]

## Steps to Reproduce

1. Clone the example repository: https://github.com/valtusovs/example-github-desktop-image-bug-2.1.0.git
2. Open History tab
3. Open the "corrupted images" commit and click on gist.jpg

### Expected Behavior

Shows the modified images next to one another, possibly indicating that the new one is broken

### Actual Behavior

The GitHub Desktop window becomes white fully.

## Additional Information

![github crash](https://user-images.githubusercontent.com/50198060/61922096-d6f1ee00-af4e-11e9-8f62-6fda488aa9dd.gif)

### Logs

```
2019-07-26T02:19:42.742Z - info: [ui] [Timing] Action 'create commit' for 'valtusovs/example-github-desktop-image-bug-2.1.0' took 1.335s
2019-07-26T02:19:46.788Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: example-github-desktop-image-bug-2.1.0 - valtusovs (has token)
2019-07-26T02:19:53.061Z - info: [ui] Executing push: git -c credential.helper= -c protocol.version=2 push origin master:master --progress (took 6.186s)
2019-07-26T02:19:56.671Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 3.526s)
2019-07-26T02:20:13.133Z - info: [ui] [AppStore] loading 6 repositories from store
2019-07-26T02:20:13.134Z - info: [ui] [AppStore] found account: valtusovs (valtusovs)
2019-07-26T02:20:13.358Z - info: [ui] [BranchPruner] Last prune took place in a day - skipping
2019-07-26T02:20:13.722Z - info: [ui] launching: 2.1.0 (Windows 10.0.17763)
2019-07-26T02:20:13.723Z - info: [ui] execPath: 'C:\Users\user_account\AppData\Local\GitHubDesktop\app-2.1.0\GitHubDesktop.exe'
```

**Comment by shiftkey** [comment]
@valtusovs thanks for the report!

I haven't had a chance to dig into the root cause but I wanted to start with confirming we can reproduce the behaviour ourselves:

On Windows 10 with our latest stable version `2.1.0` I was able to see the problem. Nothing obvious from the logs or DevTools what might be triggering the issue.

I then switched over to our `2.1.1-beta2` release (which upgrades Electron) and was able to view the problem `gist.jpg` file:

<img width="968" src="https://user-images.githubusercontent.com/359239/61951759-54862000-af88-11e9-821a-44c19f21dbe8.png">

On macOS 10.14.6 with our current beta `2.1.1-beta2` I can also view the problem file:

<img width="1093" src="https://user-images.githubusercontent.com/359239/61951184-9f069d00-af86-11e9-9844-345993f5cff4.png">

My gut feeling here is that this will resolve itself when we ship `2.1.1`, but let me know if there's any other cases you'd like us to test.
